### PR TITLE
Feature/add part attribute to today day

### PR DESCRIPTION
**Pull request by achamorro-dev**
Add part for today to cell and div to enable customisation via CSS

### CODE REVIEW

File: src/datepicker.ts [hunk 1]
```diff
@@ -901,15 +901,18 @@ export class Datepicker extends LitElement {
                       'day--today': +todayDate === curTime,
                       'day--focused': !disabled && isCurrentDate,
                     })}"
-                    part="calendar-day"
+                    .part="calendar-day${+todayDate === curTime ? ' calendar-today' : ''}"
```
Comment (motss) [inline src/datepicker.ts:904]: why using `.part` instead of `part`?

Comment (achamorro-dev) [reply src/datepicker.ts]: It was a mistake of the first try. Fixed.

**Review by motss — APPROVED** [review]
👍 LGTM

**Review by motss — APPROVED** [review]

**Comment by motss** [comment]
Merging this even though some of the tests failed. Failing tests will be resolved in separate PR.

**Pull request merged**